src/utils/azureUtils.ts

```diff
@@ -55,10 +62,17 @@ export function createAzureExtensionsGroupConfig(extensions: IAzExtMetadata[], s
     const azExtGroupConfigs: GroupNodeConfiguration[] = [];
     for (const azExt of extensions) {
         for (const azExtResourceType of azExt.resourceTypes) {
+            let iconPath: TreeItemIconPath;
```

Comment:
Logic feels a little backwards to me here.  What about doing something like:

```
let iconPath = getIconPath(resource.azExtResourceType);
if (resource.azExtResourceType === AzExtResourceType.ContainerAppsEnvironment) {
   // comment as to why this is needed
    iconPath = getIconPath(AzExtResourceType.ContainerApps);
 }

Reply:
Yeah, good call, thanks.  Been a bit frazzled today, not thinking super clearly.